### PR TITLE
Fixing maven test case

### PR DESCRIPTION
**Pull request by ScrewTSW**

### CODE REVIEW

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

File: tests/src/test/resources/arquillian_old.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Katka92) [inline tests/src/test/resources/arquillian_old.xml]: Is that file needed?

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by Katka92** [comment]
[test]
